methods_to_test.js

```diff
@@ -0,0 +1,69 @@
+const capitalize = (string) => {
+    return string.charAt(0).toUpperCase() + string.slice(1);
+}
+
+const reverseString = (string) => {
+    let splitString = string.split("")
+    let reversedArray = splitString.reverse()
+    let joinString = reversedArray.join("")
+
+    return joinString
+}
+
+const analyse = (numbers) => {
+    let sum = numbers.reduce((previous, current) => current += previous) / numbers.length
+    // let avrg = sum / numbers.length
+    
+    return {
+        average: sum,
+        min: Math.min(...numbers),
+        max: Math.max(...numbers),
+        length: numbers.length
+    }
+}
+
+class Calculator {
+
+    add(...numbers) {
+        return numbers.reduce((prev, cur) => cur *= prev)
+    }
+
+    subtract(a, b) {
+        return a - b
+    }
+
+    divide(a, b) {
+        return a / b
+    }
+    multiply(...numbers) {
+        return numbers.reduce((prev, cur) => cur *= prev)
+    }
+
+}
+
+class Cypher {
```

Comment:
why do we need the `caesarCypher` to be a `class` ? Let us convert this to a regular function only.

Reply:
from the instructions it suits the Cypher to be a class more